core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -47,6 +46,8 @@ object ConsoleConsumer extends Logging {
   var messageCount = 0
 
   private val shutdownLatch = new CountDownLatch(1)
+  private val EARLIEST_STRING = "earliest"
+  private val LATEST_STRING = "latest"
```

Comment:
Can we make the constant names more self-explanatory? e.g. `OFFSET_ARG_VAL_EARLIEST` and `OFFSET_ARG_VAL_LATEST`.